R/heatmap_mutation_frequency_bin.R

```diff
@@ -51,7 +51,7 @@
 #'
 #' @examples
 #' #load metadata.
-#' metadata = get_gambl_metadata()
+#' metadata = GAMBLR.data::gambl_metadata
```

Comment:
This function is not using the `handle_metadata` function, on purpose?

Reply:
right, yes - since this is an example, we will be sure this always works and also returns the same output (regardless of where the `repo_base` is pointing to)

Reply:
I have added the CN column to the bundled data ✅